From jm@lentin.co.uk  Fri Feb  6 15:56:23 2009
From: Jamie Lentin <jm@lentin.co.uk>
Date: Wed, 4 Feb 2009 13:38:44 +0000 (GMT)
Subject: Staging: at76_usb: Add support for OQO Model 01+
To: gregkh@suse.de
Cc: linux-kernel@vger.kernel.org
Message-ID: <alpine.DEB.2.00.0902041304560.16577@localhost.localdomain>


Add USB device ID for OQO 01+'s internal wireless LAN

An OQO employee mentions the chip's true identity here:-
   ftp://ftp.oqo.com/unsupported/linux/OQOLinux.html

Signed-off-by: Jamie Lentin <jm@lentin.co.uk>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/at76_usb/at76_usb.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/staging/at76_usb/at76_usb.c
+++ b/drivers/staging/at76_usb/at76_usb.c
@@ -231,6 +231,8 @@ static struct usb_device_id dev_table[] 
 	{USB_DEVICE(0x03eb, 0x7617), USB_DEVICE_DATA(BOARD_505A)},
 	/* Siemens Gigaset USB WLAN Adapter 11 */
 	{USB_DEVICE(0x1690, 0x0701), USB_DEVICE_DATA(BOARD_505A)},
+	/* OQO Model 01+ Internal Wi-Fi */
+	{USB_DEVICE(0x1557, 0x0002), USB_DEVICE_DATA(BOARD_505A)},
 	/*
 	 * at76c505amx-rfmd
 	 */
